customer.py

```diff
@@ -0,0 +1,150 @@
+# Helper function
+def print_stars():
+    print("\n******************************************************************************\n")
+
+# Select main menu option
+def get_main_menu_choice():
```

Comment:
This function doesn't quite fit here - I would recommend moving it back to main.py or putting it in it's own namespace file (menu.py?)